src/Meshes/mesh.ts

```diff
@@ -784,12 +796,13 @@ export class Mesh extends AbstractMesh implements IGetSetVerticesData {
     }
 
     private _sortLODLevels(): void {
+        const useScreenCoverage = this._internalMeshDataInfo._useLODScreenCoverage ? -1 : 1;

```

Comment:
I don't find the name `useScreenCoverage` very clear given how it is used... Maybe something independent from the LOD type would be better (like `resultForLessComparison`)?

Reply:
I went with `sortingOrderFactor`